src/core/heavy_keeper.cc

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include <xxhash.h>
+#include <absl/container/flat_hash_map.h>
+
+#include <cmath>
+#include <optional>
+
+#include "core/heavy_keeper.h"
+
+namespace dfly {
+
+using namespace std;
+
+uint64_t HeavyKeeper::Hash(const std::string& key) {
+    return XXH3_64bits_withSeed(key.data(), key.size(), 24061983);
```

Comment:
formatting is still off. We added presubmit checks and updated CONTRIBUTING.md on how to check yourself locally. with `pre-submit run`